### PR TITLE
docs(payment): intro

### PR DESCRIPTION
**Pull request by benjlevesque**

### CODE REVIEW

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 1]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
...
+npm install @requestnetwork/payment-processor
+# or
+yarn add @requestnetwork/payment-processor
+```
+
+
+
+## Usage
+
+### ERC20 request
```
Comment (vrolland) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]: I would put ERC20 after Ethereum as it looks more complex.

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 2]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
 
 # Introduction
 
-The Request payment library simplifies the request payments.
-
-You should now be able to create requests and to deal with basic workflow from the requester's perspective. We can ask for payments, let's make sure we can fulfill these requests now!
+In the previous sections, you have learned how to create a Request for payment. We will now explain you how to pay it.
```
Comment (ltacker) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]:
```suggestion
In the previous sections, you have learned how to create a request for payment. We will now explain to you how to pay it. 
```

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 3]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
...
-
-You should now be able to create requests and to deal with basic workflow from the requester's perspective. We can ask for payments, let's make sure we can fulfill these requests now!
+In the previous sections, you have learned how to create a Request for payment. We will now explain you how to pay it. 
 
 This section is useful if:
 * You plan to embed request payment features like a Pay button
 * You want to test the payment stage of requests you create
 
-# About the request payment library
+The payment of a Request depends on the payment network you chose:
```
Comment (ltacker) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]:
```suggestion
The payment of a request depends on the payment network you chose: 
```

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 4]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
...
+In the previous sections, you have learned how to create a Request for payment. We will now explain you how to pay it. 
 
 This section is useful if:
 * You plan to embed request payment features like a Pay button
 * You want to test the payment stage of requests you create
 
-# About the request payment library
+The payment of a Request depends on the payment network you chose: 
+
+- Address-based payment networks (available for BTC and ETH) don't have any specific requirement: any payment sent to the specify address will be considered a payment of this request. Never re-use an address!
```
Comment (ltacker) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]:
```suggestion
- Address-based payment networks (available for BTC and ERC20) don't have any specific requirement: any payment sent to the specify address will be considered a payment of this request. Never re-use an address!
```

It's ERC20 that has address-based not Ether

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 5]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
...
+- Input data payment networks (ETH only) simply requires you to specify the [Payment Reference](https://github.com/RequestNetwork/requestNetwork/blob/master/packages/advanced-logic/specs/payment-network-eth-input-data-0.2.0.md#description) in the data of the transaction.
+- Proxy payment networks (ETH and ERC20) require you to call a smart contract.
```
Comment (ltacker) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]:
```suggestion
- Proxy payment networks (ETH and ERC20) require you to call a smart contract method that forwards the payment.
```

File: packages/docs/docs/guides/4-request-payment/0-intro.md [hunk 6]
```diff
@@ -7,24 +7,87 @@ keywords: [Request payment library]
...
+
+To simplify the payment procedures of the various payment networks, you can use the dedicated library, [@requestnetwork/payment-processor](https://www.npmjs.com/package/@requestnetwork/payment-processor). 
+On top of calculating the Payment Reference and handling the transaction for you, it provides a few utilities to ensure the user meets all requirements to pay the Request (enough funds for example)
```
Comment (ltacker) [inline packages/docs/docs/guides/4-request-payment/0-intro.md]:
```suggestion
On top of calculating the Payment Reference and handling the transaction for you, it provides a few utilities to ensure the user meets all requirements to pay the request (enough funds for example)
```

**Pull request merged**